quartz-css/vhx-style-forms/docs/code/forms.html

```diff
@@ -0,0 +1,33 @@
+<!-- Standard -->
+<form class="form border radius padding-medium">
```

Comment:
One thought with the code example. Perhaps have comments throughout for clarity?

```
<form class="form border radius padding-medium">

   <!-- Standard text field -->
   <fieldset class="margin-bottom-medium">
      <label for="user_name">User name</label>
      <input type="text" name="user_name" placeholder="Enter a your user_name">
    </fieldset>

   <!-- Disabled text field -->
    <fieldset class="margin-bottom-medium">
       <label for="secret_key">Secret Key (disabled)</label>
       <input type="number" name="secret_key" disabled="" placeholder="Enter your secret key (disabled)">
   </fieldset>
     ... etc
```